lib/api-util.js

```diff
@@ -16,8 +16,11 @@ const HTTPError = sUtil.HTTPError;
  */
 function mwApiGet(app, domain, query) {
 
-    query = query || {};
-    query.continue = query.continue || '';
+    query = Object.assign({
+        format: 'json',
+        formatversion: 2,
+        continue: ''
```

Comment:
I'm not very sure about the `continue` part to be honest. Most of the API modules call the `continue` in a modulepscecific way like `blcontinue` for backlinks or `iucontinue` for imagelinks etc, so adding the `continue` by default in many cases will cause the Action API to emit a warning that it's not supported. 

However I'm not sure why it was added in the first place, maybe I'm missing something here, so cc @d00rman